packages/cli/package.json

```diff
@@ -41,7 +41,7 @@
   ],
   "dependencies": {
     "@axe-core/webdriverjs": "^4.2.0",
-    "axe-core": "^4.2.0",
+    "axe-core": "^4.2.1",
```

Comment:
Just _another_ reminder: users were able to use 4.2.1 before this patch landed. Since we have the `^` here, we'll allowing npm to install a _range_ of versions, not exclusively 4.2.1 (previously 4.2.0).